### PR TITLE
Approval screen styling updates

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://projects.engineering.redhat.com/browse/SSP-1675

- [x] Add colons to labels
- [x] Remove toggle
- [x]  Add log in button post-action
- [x] Reorder and resize buttons
- [x]  Add email to "Ordered by"
- [x]  Request ID as header
- [x]  Add spacing between rows

Old
Screen Shot 2020-08-12 at 2 17 09 PM

New
Screen Shot 2020-08-12 at 2 16 01 PM

Old
Screen Shot 2020-08-12 at 2 17 19 PM

New
Screen Shot 2020-08-12 at 2 28 48 PM

### CODE REVIEW

**Comment by epwinchell** [comment]
@dippy-bot add_reviewer @hsong-rh 
@dippy-bot add_label enhancement

**Comment by epwinchell** [comment]
@hsong-rh This is ready for review. I'm going to move the remaining tasks to another PR.

File: app/controllers/api/v1x2/stageaction/show.html.erb [hunk 1]
```diff
@@ -4,66 +4,44 @@
...
-                  var x = document.getElementById("details");
-                  if (x.style.display === "none") {
-                    x.style.display = "block";
-                  } else {
-                    x.style.display = "none";
-                  }
-              }
-          </script>
-        </section>
+        <% end %>
```
Comment (eclarizio) [inline app/controllers/api/v1x2/stageaction/show.html.erb:45]: What is the the `end` of? Maybe I missed it but I don't see any opening `do` or `if/else` or something like that.

**Comment by epwinchell** [comment]
> Just the one comment, will approve & merge after either an adjustment or a good answer 👍

@eclarizio I cleaned up the indents. Here's the "do" https://github.com/RedHatInsights/approval-api/blob/8335a61361361b68e4224acee7b58171c6f46f4a/app/controllers/api/v1x2/stageaction/show.html.erb#L32

**Pull request merged**